src/doc/book/getting-started.md

```diff
@@ -164,11 +164,15 @@ installed. Doing so will depend on your specific system, consult its
 documentation for more details.
 
 If not, there are a number of places where we can get help. The easiest is
+[the #rust-beginners IRC channel on irc.mozilla.org][irc-beginners], which we
+can access through [Mibbit][mibbit-beginners] and for general discussion
 [the #rust IRC channel on irc.mozilla.org][irc], which we can access through
```

Comment:
This second clause of "which we can access through mibbit" can probably be dropped as it was already mentioned before, otherwise looks good to me!